plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -33,6 +33,10 @@ open class KtlintExtension(
      * Enable console output mode.
      */
     val outputToConsole: Property<Boolean> = objectFactory.property { set(true) }
+    /**
+     * Enabled colored output to console.
+     */
+    val coloredOutput: Property<Boolean> = objectFactory.property { set(false) }
```

Comment:
Why default to false? I'm good either way on this one, but why make this the default?

Reply:
Yes, I think default value as `true` will be better, as on plain console type it will not be active.